### ISSUE TITLE
Train model problem

### ISSUE THREAD

**Issue by Cun-bao** [ydhongHIT/DDRNet#33]
I train the DDRNet_23_silm and use the pretrained model but I get a wrong result, the mIoU only 58%, too low than your result in paper.I also find the loss  decline is slow,  the best loss is 0.9. 
I use a single GPU, lr:3e-3, epochs:500, optim:SGD, loss:ohemceloss
